src/zvknhk.adoc

```diff
@@ -0,0 +1,435 @@
+
+[[zvknhk,zvknhk]]
+== "Zvknhk" Extension for Vector Keccak 
+
+This chapter describes the `Zvknhk`  Vector Cryptography extension, which adds vector instructions to compute the Keccak sponge function using in NIST SHA3 and numerous Post Quantum Cryptography (PQC) algorithms.
+
+
+This document is in _Development_ state.
+Assume that everything can change.
+This document has not been publicly reviewed nor approved yet.
+For more information, see link:https://lf-riscv.atlassian.net/wiki/spaces/HOME/pages/16154861/RISC-V+Specification+Status[here].
+
+
+[[zvknhk-overview]]
+=== Extension Overview
+
+This section introduces `Zvknhk`.
+
+`Zvknhk` depends on Zve64x.
+
+
+As the instructions defined in this specification might be used to implement cryptographic primitives,
+they must be implemented with data-independent execution latencies as defined in 
+link:https://github.com/riscv/riscv-isa-manual/blob/main/src/scalar-crypto.adoc#data-independent-execution-latency-subset-zkt.
```

Comment:
Zvkt right?

Reply:
We use Zkt as the reference for the data independent execution latency definition (this is basically a copy/paste from the vector crypto spec, I kept the same reference).